tests/test_topology.py

```diff
@@ -352,6 +352,16 @@ def test(self):
     located = sample.eval(self.geom)
     self.assertAllAlmostEqual(located, target)
 
+  def test_maxdist(self):
+    if self.etype == 'square' and self.mode != 'nonlinear':
+      raise unittest.SkipTest('maxdist does not affect rectilinear meshes')
```

Comment:
With `unittest.SkipTest` this test will be marked as skipped, which we usually only do when dependencies are missing. Maybe you can replace this with

```python
@testing.enable_if(lambda etype, mode, **kwargs: etype != 'square' or mode != 'nonlinear')
def test_maxdist(self):
  target = numpy.array([(.2,.3), (.1,.9), (0,1)])
  ...
```